modules/net-vpc-swp/variables.tf

```diff
@@ -62,6 +74,7 @@ variable "policy_rules" {
       action                 = optional(string, "ALLOW")
       enabled                = optional(bool, true)
       tls_inspection_enabled = optional(bool, false)
+      description            = optional(string, "Managed by Terraform.")
```

Comment:
I would use `optional(string)` and then internally use the top-level `description` variable if this is null.

Reply:
Good point, can do.